project.clj

```diff
@@ -14,4 +14,8 @@
                     [org.slf4j/jul-to-slf4j "1.7.7"]
                     [org.slf4j/jcl-over-slf4j "1.7.7"]
                     [org.slf4j/log4j-over-slf4j "1.7.7"]
-                    [criterium "0.4.3"]]}})
+                    [criterium "0.4.3"]]}}
+  :repositories {"farmlogs-internal"
```

Comment:
In fact after looking at this for a bit, I'd say the value of setting up Sonatype for publishing `SNAPSHOTS` is pretty low. I'm going to leave the FL internal repo here.